### ISSUE TITLE
How to disable sidecar injection in a particular pod?

### ISSUE THREAD

**Issue by saranshkarira** [k8tz/k8tz#112]
One of my pod runs on third party image that is very small, doesn't even have /etc and is crashing due to injection. any annotation that can help me disable injection for this particular pod/deployment?

**Comment by yonatankahana** [comment]
yes, there are [annotations](https://github.com/k8tz/k8tz?tab=readme-ov-file#annotations), you can annotate your pod with `k8tz.io/inject: "false"` to disable k8tz injection on that specific pod (or even namespace)
